pkg/cmd/kafka/consumergroup/resetoffset/reset_offset.go

```diff
@@ -133,8 +133,11 @@ func runCmd(opts *options) error {
 		return err
 	}
 
+	offsetEnum, _ := kafkainstanceclient.NewOffsetTypeFromValue(opts.offset)
```

Comment:
Not sure if we should handle error as we are doing validation before so we are guaranteed to have valid offset.

Reply:
We can skip handling here.